app/microservice_views.py

```diff
@@ -37,7 +37,7 @@ def create_url(self, url, database_name, clear):
     if url == None:
       url = (f'/microservices/{database_name}{clear}')
     else:
-      url = (f'/microservices/{url}{clear}')
+      url = (f'{url}')
```

Comment:
Does this do anything?

In python, ('a') normally produces a single value 'a', not a tuple. If you need a tuple, you need a trailing comma: ('a',)

If you only want a string, the else branch would appear to be doing nothing.

Reply:
Ahh, was expecting a string, the brackets must have been copied over from elsewhere erroneously. 
The else branch should set the URL  to whatever the value has been set to (if it was to be overridden  from the default  format above)